docs/source/models/concept_models/vivarium_nutrition_optimization/kids/concept_model.rst

```diff
@@ -1422,9 +1454,60 @@ Wave III
   * - Model
     - Outputs
     - Overall strata
-  * - 14
-    - 
-    - 
+  * - 14.0
+    - 1. Deaths and YLLs (cause-specific)
+      2. YLDs (cause-specific)
+      3. Cause state person time
+      4. Cause state transition counts
+      5. Stunting state person time, stratified by antenatal intervention coverage
+      6. Wasting state person time, stratified by antenatal intervention coverage
+    - * Age group
+      * Sex
+  * - 15.0
+    - 1. Deaths and YLLs (cause-specific)
+      2. YLDs (cause-specific)
+      3. Cause state person time
+      4. Cause state transition counts
+      5. Stunting state person time, stratified by antenatal intervention coverage
+      6. Wasting state person time, stratified by antenatal intervention coverage
+    - * Age group
+      * Sex
+      * Subnational location
+  * - 16.0
+    - 1. Deaths
+      2. Wasting state person time, stratified by SQ-LNS treatment coverage
+      3. Stunting state person time, stratified by SQ-LNS treatment coverage
+    - * Age group
+      * Sex
+      * Subnational location
```

Comment:
I was thinking that we could eventually stratify just by "targeted or not" rather than subnational location, which will hopefully make our observers run faster - I guess that's equivalent to what you included as "SQLNS treatment coverage", right? Maybe we can verify in this model that the coverage is as we expect it to be by subnational location and then we can drop the subnational stratification moving forward to increase speed/efficiency